sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample4_RecognizeCustomFormsFromUri.cs

```diff
@@ -14,18 +15,25 @@ namespace Azure.AI.FormRecognizer.Samples
     public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
     {
         [Test]
-        [Ignore("Need to revisit how to pass the modelId. Issue https://github.com/Azure/azure-sdk-for-net/issues/11493")]
         public async Task RecognizeCustomFormsFromUri()
         {
             string endpoint = TestEnvironment.Endpoint;
             string apiKey = TestEnvironment.ApiKey;
+            string trainingFileUrl = TestEnvironment.BlobContainerSasUrl;
+
+            // Firstly, create a trained model we can use to recognize the custom form.
+
+            FormTrainingClient trainingClient = new FormTrainingClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+            CustomFormModel model = await trainingClient.StartTraining(new Uri(trainingFileUrl)).WaitForCompletionAsync();
+
+            // Proceed with the custom form recognition.
 
             FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 
             string formUri = FormRecognizerTestEnvironment.CreateUri("Form_1.jpg");
-            string modelId = "<your model id>";
+            string modelId = model.ModelId;
```

Comment:
we could actually include the `modelId` into the snippet, similar to what we do here: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/SampleSnippets.cs#L22

by setting `//@@ string modelId = "<modelId >";`

Reply:
Ohh, this is nice. Added it to the `FormRecognizerRecognizeCustomFormsFromFile` snippet as well.